web/src/components/organisms/Asset/Asset/hooks.ts

```diff
@@ -35,15 +38,13 @@ export default (assetId?: string) => {
           refetchQueries: ["GetAsset"],
         });
         if (result.errors || !result.data?.updateAsset) {
-          // TODO: notification
-          // console.log(t("Failed to update asset."));
+          Notification.error({ message: t("Failed to update asset.") });
         }
         if (result) {
-          // TODO: notification
-          // console.log(t("Asset was successfully updated."));
+          Notification.success({ message: t("Asset was successfully updated.") });
```

Comment:
I'll stop fixing these, so please check your translation files and find all areas where the notificaiton style doesn't match the other notifications (Not using exclamation marks, word order is different, etc)

Reply:
Done!